packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -93,6 +94,46 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     blockWhileInProgress(inspectorController.collapseDetailsToSelected);
   }
 
+  Widget _buildErrorList(
+    BuildContext context,
+    List<InspectableWidgetError> errors,
+  ) {
+    return ListView(
+      padding: const EdgeInsets.all(8),
+      children: <Widget>[
+        for (final error in errors)
+          Padding(
+            padding: const EdgeInsets.only(bottom: 8),
+            // TODO(dantup): How to make the cursor a hand pointer?
+            child: GestureDetector(
```

Comment:
try using an InkWell widget instead of a gesture detector

Reply:
Aha, thanks! (Done in 6f779c1)